db/re/item_combos.yml

```diff
@@ -3085,12 +3085,11 @@ Body:
       bonus bIntravision;
   - Combos:
       - Combo:
-          - Giant_Protection
-          - Giant_Axe
+          - Giant_Protection     # 28521
```

Comment:
Why would we want to re-add Item IDs?

Reply:
I asked him to comment the ID when he adds a combo. It's easier for example to search the item (and combo) on divine pride, and it doesn't hurt the readability/loading

Reply:
Divine pride supports searching for [Giant_Protection ](https://www.divine-pride.net/database/item?Name=Giant_Protection+&Description=&function=&find=Search) and [Giant_Axe.](https://www.divine-pride.net/database/item?Name=Giant_Axe&Description=&function=&find=Search)
And we removed it for maintainability. X_x

Reply:
It's faster to copy/past the id in the url. The comment can be removed anytimes it's just an extra